tasks/agent6-linux.yml

```diff
@@ -54,14 +54,22 @@
 
 - name: Ensure datadog-agent is running
   service:
-    name: datadog-agent
+    name: "{{ item }}"
```

Comment:
Issue here is that `datadog-agent-process` and `datadog-agent-trace` might not be configured to be running. The trace agent is enable by default but can be disable from the configuration and the process agent is disable by default on most OS.

Checking that those process are running by default might flag a a successful install as failed.

We should ensure they're running only when enabled in the configuration.

Reply:
Hi @hush-hush, as already mentioned in https://github.com/DataDog/ansible-datadog/pull/153#issuecomment-430397578, the current situation is that the trace and process agents' systemd services are already always enabled, no matter the datadog config passed to these agents.

I can remove this change to avoid enabling them if some other process somehow disables them.

The alternative I can think of right now would be to introduce new ansible parameters `datadog_process_enabled` and `datadog_trace_enabled` next to `datadog_enabled` in https://github.com/DataDog/ansible-datadog/blob/7b209f3e6300fd3c33d3a4f0a2c2c59e91a8c790/defaults/main.yml#L2

This kind of feel out of scope of this PR though.

WDYT?

Reply:
I've implemented option 1, i.e. do not enable all datadog agent v6 services, only care about datadog-agent.

Reply:
Adding new ansible parameters for each agents seems like a good solution but is outside the scope of this PR.

Thanks for the update !